api/api/src/main/scala/uk/ac/wellcome/platform/api/services/RelatedWorkRequestBuilder.scala

```diff
@@ -0,0 +1,106 @@
+package uk.ac.wellcome.platform.api.services
+
+import com.sksamuel.elastic4s.ElasticDsl._
+import com.sksamuel.elastic4s.Index
+import com.sksamuel.elastic4s.requests.searches.{MultiSearchRequest, SearchRequest}
+import com.sksamuel.elastic4s.requests.searches.queries.Query
+
+case class RelatedWorkRequestBuilder(index: Index,
+                                     path: String,
+                                     maxRelatedWorks: Int = 1000) {
+
+  // To reduce response size and improve Elasticsearch performance we only
```

Comment:
Not sure this improves Elasticsearch performance, but will reduce data across the network, in turn reducing latency.

Reply:
Agree that will be there bigger issue, but reading less bytes from the disk / memory will result in a quicker response time.

Reply:
Indeed, just thought the comment should point to less bytes, not real ES perf... maybe they're the same though.